AssetManagementTestProject/DAO/CreateUserDAO.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace AssetManagementTestProject.DAO;
+
+public class CreateUserDAO
+{
+    public partial class CreateUserRequest
+    {
+        public string FirstName { get; private set; }
+        public string LastName { get; set; }
+        public string DateOfBirth { get; private set; }
+        public int Gender { get; set; }
+        public string JoinedDate { get; private set; }
+        public int Role { get; private set; }
+        public int Location { get; private set; }
+
+        public CreateUserRequest(string firstName, string lastName, string dateOfBirth, int gender, string joinedDate, int role, int location)
+        {
+            FirstName = firstName;
+            LastName = lastName;
+            DateOfBirth = dateOfBirth;
+            Gender = gender;
+            JoinedDate = joinedDate;
+            Role = role;
+            Location = location;
+        }
+    }
+
+    public partial class CreateUserResponse
+    {
+        public bool IsSuccess { get; private set; }
+        public string Message { get; private set; }
+        public Data Data { get; private set; }
+
+        public CreateUserResponse(bool isSuccess, string message, Data data)
```

Comment:
Param should be a noun, not an expectation

Reply:
Done